python/composio/tools/base/runtime.py

```diff
@@ -273,6 +273,12 @@ def _build_executable_from_args(  # pylint: disable=too-many-statements
 ) -> t.Tuple[t.Callable, t.Type[BaseModel], t.Type[BaseModel], bool]:
     """Build execute action from function arguments."""
     argspec = inspect.getfullargspec(f)
+    missing_annot = set(argspec.args) - set(argspec.annotations)
+    if len(missing_annot) > 0:
+        raise InvalidRuntimeAction(
+            message=f"Following arguments are missing type annotations: {missing_annot}"
```

Comment:
Consider making the error message more actionable by including a suggestion for how to fix it. For example:
`message=f"Type annotations are required for all parameters. Missing annotations for: {missing_annot}. Add type hints like 'param: type' for each parameter."`